wqu_app/message.py

```diff
@@ -14,8 +14,8 @@ def get_geolocation(ip_address):
     data = response.json()
     city = data['city']
     coords = [float(coord) for coord in data['loc'].split(',')]
-
-    return coords, city
+    region = data['region']
```

Comment:
Would it make sense to move this line right below Line 15? Also would it be better to combine city and region into one variable?